conans/client/cmake.py

```diff
@@ -52,10 +53,11 @@ def _generator(self):
                 return base
 
         if operating_system == "Windows":
-            if compiler == "gcc":
-                return "MinGW Makefiles"
-            if compiler in ["clang", "apple-clang"]:
-                return "MinGW Makefiles"
+            non_msvc = (compiler != "Visual Studio")
```

Comment:
Michele, sorry for the trouble, but maybe one of my previous comments was lost.
``compiler`` will always be different to "Visual Studio" at this point, as the previous ``if`` will always return.